samples/sensor/mcux_acmp/src/main.c

```diff
@@ -28,13 +34,17 @@ struct acmp_attr {
 };
 
 static const struct acmp_attr attrs[] = {
+#if MCUX_ACMP_HAS_INPSEL
```

Comment:
Is this change needed. If so, then a similar change should be made to the driver and include files

Reply:
If we set the unsupported attribute the mcux_acmp_attr_set will return -ENOTSUP error. So there are two potential resolutions:
1. Did not provide the attribute which not supported, this is what I did now, but need also made the change to the driver include files. 
2. Ignore the -ENOTSUP error in the application. 
I'm not sure which should be preferred.